app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash[:success] = t "users.new.flash_success"
+      redirect_to @user
+    else
+      render :new
```

Comment:
thêm flash cho else nữa e

Reply:
trong view new có in lỗi của form ra từ mảng user.errors nên code trong sách cũng không cần dùng flash ạ